### ISSUE TITLE
Instance Button Bar Inconsistency between instance state / volumes

### ISSUE THREAD

**Issue by scottsignal** [apache/cloudstack#9495]
##### ISSUE TYPE

* Improvement Request

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.19.1.0
~~~

##### OS / ENVIRONMENT

Ubuntu 22.04
KVM

##### SUMMARY

1.)  The button bars between instance states (Powered on vs Powered off) seem to be inconsistent.  Some items are greyed out or missing and some items use different icons across different screens.

2.) Volume Snapshot uses the wrong button Ant icon in the GUI.  It's using the Instance Snapshot icon.

##### STEPS TO REPRODUCE

~~~
1.) Compare the icon bars between a powered off instance and a powered on instance
2.) Compare the "Volume Snapshot" icons on a Volume vs. Powered on Instance.
~~~

##### EXPECTED RESULTS

~~~
1.) Seems like there should be some consistency.  Either the icons are greyed out if the function is not available or they should be missing. 

2.) The icon used when taking a Volume Snapshot from **Storage --> Volumes** seems like it should be the solid filled in camera, not the outline camera.
~~~

##### ACTUAL RESULTS

1.) View Console/Copy URL are greyed out when powered off and the scale/change service offering greyed out when powered on.  The opposite behavior occurs on other icons such as snapshots which are just missing instead of being greyed out from being powered off.

Powered off instance:
![image](https://github.com/user-attachments/assets/666cc725-6641-4623-a97d-99b847b72033)

Powered on instance:
![image](https://github.com/user-attachments/assets/51bdd77c-d864-4b74-b292-139611590336)

2.) Volume Snapshots is using the outline camera icon:
![image](https://github.com/user-attachments/assets/cdb9773e-5b17-42f2-a25d-d69422fc99e2)

A powered on instance uses the solid camera icon for Volume snapshots on an instance.  
![image](https://github.com/user-attachments/assets/26b13ed7-1d72-44e0-8b8b-0937550a4407)

**Comment by rajujith** [comment]
> 1.) The button bars between instance states (Powered on vs Powered off) seem to be inconsistent. Some items are greyed out or missing and some items use different icons across different screens.

@scottsignal  Are you suggesting that either it should be greyed out or it should not be visible based on the power status of the instance instead of mixing both grey out and removing the icons?

**Comment by scottsignal** [comment]
> > 1.) The button bars between instance states (Powered on vs Powered off) seem to be inconsistent. Some items are greyed out or missing and some items use different icons across different screens.
> 
> @scottsignal Are you suggesting that either it should be greyed out or it should not be visible based on the power status of the instance instead of mixing both grey out and removing the icons?

Yes.  Really it almost feels like the instance/volume snapshot icons should be there and greyed out when an instance is powered off.  We had some feedback that it was confusing.
